theme/less/_header.less

```diff
@@ -8,15 +8,17 @@ header > p
   color: #7d7d7d;
```

Comment:
Differences in mobile:
- line-height and font-sizes
  - heading
  - Sub heading
  - article heading (line-height)